php.md

```diff
@@ -0,0 +1,41 @@
+# PHP
+
+
+## Prerequisite knowledge:   
+HTML
+
+
+## Classification / Acceptance :
+
+#### Apprentice aka _Hello World_
+ * what kind of language PHP is?
+ * explain the process of generating HTML code ‘by’ PHP script
+ * enumerate at least 6 basic data types used in PHP
+ * create conditional statements and loops (loop through indexed and associative arrays)
+ * create your own function
+ * create your own class
+ * name at least 3 superglobal variables
+
+#### Intermediate aka _It means i can really use it_
+
+ * use require() and include() directives to use contents of other PHP files
+ * read/write data from/to a file
+ * connect to external database using PDO
+ * handle possible exceptions where necessary
+
+
+## Sources:
+
+For those who like documentation-like tutorials
+[Brief and shallow description of php](https://www.w3schools.com/php/)
+
+
+For those who like learning from books:
+[Luke Welling, Laura Thomson, “PHP i MySQL. Tworzenie stron WWW. Vademecum profesjonalisty”](http://miroslawzelent.pl/ksiazki-warte-uwagi/)
```

Comment:
I think we should avoid sources in PL language. The idea of BIT is to be used globally so EN is the most reasonable choice. Please link to the EN version of the book.